Bugzilla/Elastic/Indexer.pm

```diff
@@ -213,7 +155,8 @@ sub _select_updated_ids {
 sub bulk_load_ids {
     my ($self, $class, $ids) = @_;
 
-    $self->put_mapping($class);
+    $self->_create_index($class);
```

Comment:
It would be nice if the commit message mentioned this, since no other bug exists documenting the bug.